src/chromium/backgroundOptions.js

```diff
@@ -0,0 +1,139 @@
+const defaultOptions = {
+  encryptKey: {
+    title: 'Store keys encrypted using master password',
+    info: 'Enabling this option will encrypt your secret key using your master password. This is neccessary for enabling Chrome sync storage.',
+    type: 'boolean',
+    default: false,
+    value: false,
+    disabled: (options) => {
+      return options.masterPassword.value.length == 0;
```

Comment:
`Error in event handler for runtime.onMessage: TypeError: Cannot read property 'value' of undefined` Got this when I generated a new key.

Reply:
Could I get steps to reproduce? Was unable to reproduce by

1. Loading extension on fresh Chrome
2. Navigating to https://twitter.com
3. Opening dev tools
4. Generating new secret keys

Icon update functionality remained as well, which leads me to believe this is the root of the #11

Reply:
Did you mess with updateOptions() or the options in any way during testing?

Reply:
I could not reproduce this error. We should address this at a later time if it ever shown to be reproducible.